.travis.yml

```diff
@@ -9,4 +9,5 @@ before_script:
   - yarn run db:reset
 script:
   - yarn test
+  - yarn run enforce
```

Comment:
nit: you don't need `run` after yarn commands. you can change these to `yarn enforce` and `yarn lint`

Reply:
done! ty